incident.go

```diff
@@ -0,0 +1,156 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	TEAM_CUSTOM_FIELD_ID = "01G49Y5JP7RM5Q3CR02G2EQE1Q"
```

Comment:
For reusability and testability this should be configuration and not a constant. But no hard feelings due to the current use case. Separate exporters for the tools would have been better so other people can reuse them individually. But all good for now.

Reply:
totally agree :D